profile/src/main/kotlin/com/imashnake/animite/profile/ProfileScreen.kt

```diff
@@ -200,12 +208,82 @@ fun UserTabs(modifier: Modifier = Modifier) {
                     )
                 )
         ) {
-            Text(
-                text = stringResource(coreR.string.coming_soon),
-                color = MaterialTheme.colorScheme.onSurfaceVariant,
-                style = MaterialTheme.typography.labelLarge,
-                modifier = Modifier.align(Alignment.Center)
-            )
+            // TODO: Use `AnimatedContent`.
+            when (state) {
+                ProfileTabs.ABOUT -> AboutTab(viewer)
+                else -> Text(
+                    text = stringResource(coreR.string.coming_soon),
+                    color = MaterialTheme.colorScheme.onSurfaceVariant,
+                    style = MaterialTheme.typography.labelLarge,
+                    modifier = Modifier.align(Alignment.Center)
+                )
+            }
+        }
+    }
+}
+
+enum class ProfileTabs(val title: String) {
+    ABOUT("About"),
+    ANIME("Anime"),
+    MANGA("Manga"),
+    FAVOURITES("Fave"),
+    STATISTICS("Stats")
+}
+
+@Composable
+fun AboutTab(
+    viewer: Viewer,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier.padding(
+            top = LocalPaddings.current.large,
+            start = LocalPaddings.current.large,
+            end = LocalPaddings.current.large
+        ),
+        verticalArrangement = Arrangement.spacedBy(LocalPaddings.current.medium)
+    ) {
+        Genres(viewer.genres)
+    }
+}
+
+@Composable
+fun Genres(
+    genres: List<Viewer.Genre>,
+    modifier: Modifier = Modifier
+) {
+    Text(
+        text = "Genres",
+        style = MaterialTheme.typography.titleMedium
+    )
+    Row(modifier.height(IntrinsicSize.Max)) {
+        Column(horizontalAlignment = Alignment.End) {
+            genres.forEach {
+                Text(
+                    text = it.genre,
+                    style = m3TextStyles().textStyle.copy(
+                        color = MaterialTheme.colorScheme.onBackground.copy(alpha = 0.74f)
+                    )
+                )
+            }
+        }
+        Column(
+            modifier = Modifier
+                .fillMaxHeight()
+                .padding(start = LocalPaddings.current.small),
+            verticalArrangement = Arrangement.spacedBy(LocalPaddings.current.tiny)
+        ) {
+            val highestCount = genres.maxOf { it.mediaCount }
+            genres.forEach {
+                val weight = it.mediaCount/highestCount.toFloat() - 0.2f
+                Surface(
```

Comment:
Might want to replace `Surface` with a regular `Box` / `Spacer` since you're not using the `content` lambda here